README.md

```diff
@@ -1,2 +1,51 @@
-# annorxiver
-Annotating Rxiv preprints in an automated fashion
+# Annorxiver
+
+## Citation
+- [Manuscript in progress](greenelab.github.io/annorxiver_manuscript)
+- doi and full citation coming soon!!
+
+## About
+This repository contains code for the annorxvier project.
+This project focuses on analyzing the linguistic style and content of bioRxiv preprints and understanding how these features change when preprints undergo the publication process.
+We are currently working on a manuscript for the work in this repository.
+Feel free to check it out! (link above).
+We also created a [web app](greenelab.github.io/preprint-similarity-search) allows anyone to input a bioRxiv or medRxiv preprint and receive a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.
+
+## Directory Structure
+| Folder/file | Description |
+| --- | --- | 
+| [annorxiver_modules](annorxiver_modules) | This folder contains universal functions that other notebooks in this repository will use |
+| [biorxiv](biorxiv) | This folder contains all experiments that are related to biorxiv preprints. | 
+| [credentials](credentials) | Some of the code in this repository needs credentials to run. Any users that need to run those notebooks use check this folder first. |
```

Comment:
```suggestion
| [credentials](credentials) | Some of the code in this repository need credentials to run. Any user that needs to run those notebooks should check this folder first. |
```

Reply:
Do all notebooks need a user to fill out credentials? I assume this just pertains to the download of the data. If so, maybe you could add a separate section for how to run the experiments.